### PR TITLE
XRDDEV-1746 system settings view

### PR DESCRIPTION
**Pull request by Riippi**
https://jira.niis.org/browse/XRDDEV-1746

Implements the system settings view

Might need https://github.com/nordic-institute/X-Road/pull/1022 to be merged. Otherwise some icons might not show.

### CODE REVIEW

File: src/centralserver/admin-ui/src/views/Settings/SystemSettings/SystemSettings.vue [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+      <v-card class="pb-4" flat>
+        <div class="card-top">
+          <div class="card-main-title">
+            {{ $t('systemSettings.managementServices') }}
+          </div>
+        </div>
+
+        <table class="xrd-table mt-0 pb-3">
+          <tbody>
+            <!-- SOFTWARE token table body -->
```
Comment (carohauta) [inline src/centralserver/admin-ui/src/views/Settings/SystemSettings/SystemSettings.vue:92]: Software token?

File: src/centralserver/admin-ui/src/views/Settings/SystemSettings/SystemSettings.vue [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+              </tr>
+              <tr>
+                <td>
+                  <div>
+                    <div>
+                      {{ $t('systemSettings.centralServerAddress') }}
+                    </div>
+                  </div>
+                </td>
+                <td>DEV</td>
```
Comment (carohauta) [inline src/centralserver/admin-ui/src/views/Settings/SystemSettings/SystemSettings.vue:67]: Maybe make all the tables dynamic already in the mock. It should give more value to the mocks and makes it quicker to hook them up with backend later on.

File: src/centralserver/admin-ui/src/views/Settings/SystemSettings/SystemSettings.vue [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+      <template #footer>
+        <div class="custom-footer"></div>
+      </template>
+    </v-data-table>
+  </sub-view-container>
+</template>
+
+<script lang="ts">
+/**
+ * View for 'trust services' tab
```
Comment (carohauta) [inline src/centralserver/admin-ui/src/views/Settings/SystemSettings/SystemSettings.vue:232]: Trust services?

**Review by carohauta — CHANGES_REQUESTED** [review]

File: src/centralserver/admin-ui/src/views/Settings/SystemSettings/SystemSettings.vue (hunk 1 above)
Comment (Riippi) [reply src/centralserver/admin-ui/src/views/Settings/SystemSettings/SystemSettings.vue]: Fixed

File: src/centralserver/admin-ui/src/views/Settings/SystemSettings/SystemSettings.vue (hunk 2 above)
Comment (Riippi) [reply src/centralserver/admin-ui/src/views/Settings/SystemSettings/SystemSettings.vue]: Fixed

File: src/centralserver/admin-ui/src/views/Settings/SystemSettings/SystemSettings.vue (hunk 3 above)
Comment (Riippi) [reply src/centralserver/admin-ui/src/views/Settings/SystemSettings/SystemSettings.vue]: Fixed

**Review by carohauta — APPROVED** [review]
🦄

**Comment by carohauta** [comment]
Waiting for checks to finish

**Pull request merged**